src/BaGet.Core/Authentication/AuthenticationHeaderValueExtensions.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Net;
+using System.Net.Http.Headers;
+using System.Text;
+
+namespace BaGet.Core.Authentication
+{
+    public class AuthenticationHeaderValueExtensions
+    {
+        public static string GetBase64BasicAuthHeaderString(NetworkCredential credential)
+        {
+            if (credential == null) throw new ArgumentNullException(nameof(credential));
+            return System.Convert.ToBase64String(System.Text.ASCIIEncoding.ASCII.GetBytes(GetBasicAuthHeaderString(credential)));
+        }
+
+        public static string GetBasicAuthHeaderString(NetworkCredential credential)
```

Comment:
Only `GetBase64BasicAuthHeaderString` is supposed to be called by the authentication pipeline, right? If so, let's make this method private.